core/src/main/resources/log4j4ibis.xml

```diff
@@ -123,7 +127,7 @@
 		<Logger name="nl.nn.testtool" additivity="false" level="INFO">
 			<AppenderRef ref="ladybug"/>
 		</Logger>
-		<Logger name="nl.nn.ibistesttool" additivity="false" level="INFO">
+		<Logger name="org.frankframework.ibistesttool" additivity="false" level="INFO">
```

Comment:
Previously, I tried to change this as well, in a PR. But I got rejected by other members of the Core team 😉 
The argument was that these packages are targeted at the external `Ladybug`.

Reply:
No, nl.nn.testtool is.